server/src/main/resources/schema/jboss-deployment-structure-1_3.xsd

```diff
@@ -0,0 +1,778 @@
+            xmlns="urn:jboss:deployment-structure:1.3"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="1.3">
+
+    <!-- Root element -->
+    <xsd:element name="jboss-deployment-structure" type="jbossStructureType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Root element for a jboss-structure file.
+            </xsd:documentation>
+        </xsd:annotation>
+    </xsd:element>
+
+    <xsd:complexType name="jbossStructureType">
+        <xsd:annotation>
+            <xsd:documentation>
+                The jboss structure declaration type; contains deployments and additional modules.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:sequence minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="ear-subdeployments-isolated" type="xsd:boolean" minOccurs="0" maxOccurs="1">
+                <annotation xmlns="http://www.w3.org/2001/XMLSchema">
+                    <documentation>
+                        Flag indicating whether each of the subdeployments within a .ear can access classes belonging to
+                        another subdeployment within the same .ear. Setting this to false, allows the subdeployments to
+                        see classes belonging to other subdeployments within the .ear.
+                        For example:
+                        myapp.ear
+                        |
+                        |--- web.war
+                        |
+                        |--- ejb1.jar
+                        |
+                        |--- ejb2.jar
+
+                        If the ear-subdeployments-isolated is set to false, then the classes in web.war can access
+                        classes
+                        belonging to ejb1.jar and ejb2.jar. Similarly, classes from ejb1.jar can access classes from
+                        ejb2.jar
+                        (and vice-versa).
+
+                        *Note that this flag, has no effect on the isolated classloader of the .war file(s). i.e.
+                        irrespective
+                        of whether this flag is set to true or false, the .war within a .ear will have a isolated
+                        classloader
+                        and other subdeployments within that .ear will not be able to access classes from that .war.
+                        This is
+                        as per spec*
+                    </documentation>
+                </annotation>
+            </xsd:element>
+            <xsd:element name="ear-exclusions-cascaded-to-subdeployments" type="xsd:boolean" minOccurs="0" maxOccurs="1">
+                <annotation xmlns="http://www.w3.org/2001/XMLSchema">
+                    <documentation>
+                        Flag indicating whether .ear exclusions defined in the root deployment should be cascaded to
+                        sub-deployments.
```

Comment:
This only applies to the "exclusions" element in deploymentType and not to the "exclude-subsystems". Perhaps say "Flag indicating whether .ear module exclusions..."?

If someone reads the exclude-subsystems doc, its clear that this wouldn't apply, but it took me a while to figure that out.